enzyme/test/Enzyme/ReverseMode/blas/cblas_daxpy_a.ll

```diff
@@ -0,0 +1,26 @@
+;RUN: %opt < %s %loadEnzyme -enzyme -mem2reg -instsimplify -simplifycfg -S | FileCheck %s
+
+define void @wrapper(i32 %n, double %a, double* %x, double* %y) {
+entry:
+  tail call void @cblas_daxpy(i32 %n, double %a, double* %x, i32 1, double* %y, i32 1)
+  ret void
```

Comment:
Can you pass in the incx and incy as variables?

Reply:
Did you mean defining incx, incy as local variables in `wrapper()`? Why we need to do that?

Reply:
or you mean adding incx and incy to the arguments of `wrapper()`? In this case it will be the same as `cblas_daxpy` and we probably don't need a wrapper at all

Reply:
I am a little bit confused because in the previous `ddot` test cases the wrapper function accepts pointers only, and integers are also passed as literals so I assume that was fine

Reply:
Yes (and it would be wise to update the ddot test as well, though that doesn't need to be this PR).

Reply:
Oh you are right making them variables indeed help us find more bugs (the variables need to be `getNewFromOriginal`'d before using malloc/memcpy)

Reply:
I would also update the other blas tests as well if you have a chance to check for the same issue.